ocaml/forkexecd/lib/fe_systemctl.ml

```diff
@@ -130,6 +130,15 @@ let is_active ~service =
   in
   Unix.WEXITED 0 = status
 
+let is_enabled ~service =
```

Comment:
This function mostly duplicate with `is_active`, refine to abstract a common function?